RELEASE_NOTES.md

```diff
@@ -89,9 +89,9 @@ changes (where available).
   (albeit in a more limited way).
 
 - The default workflow configuration option (preferences > processing
-> auto-apply pixel workflow defaults) has now been adjusted to
-incorporate the previous chromatic adaptation workflow setting and to
-include the new sigmoid module.  Options are now as follows:
+  > auto-apply pixel workflow defaults) has now been adjusted to
+  incorporate the previous chromatic adaptation workflow setting and to
+  include the new sigmoid module.  Options are now as follows:
 
   - scene-refrerred (filmic) -- default
```

Comment:
tyop

Reply:
what's the problem? feel like I'm being blind

Reply:
ok I see it now :)